Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -1096,7 +1098,18 @@ + (NSData *)getClientDataJSONForAuthenticationType:(_WKWebAuthenticationType)typ
     RetainPtr<NSData> clientDataJSON;
 
 #if ENABLE(WEB_AUTHN)
-    clientDataJSON = produceClientDataJson(type, challenge, origin);
+    clientDataJSON = produceClientDataJson(type, challenge, origin, WTF::nullString(), false);
+#endif
+
+    return clientDataJSON.autorelease();
+}
+
++ (NSData *)getClientDataJSONWithTopOrigin:(_WKWebAuthenticationType)type challenge:(NSData *)challenge origin:(NSString *)origin topOrigin:(NSString *)topOrigin crossOrigin:(bool)crossOrigin
```

Comment:
Any reason the implementation cannot figure out the scope using topOrigin and origin? Why do we need the crossOrigin parameter?